api/src/test/java/ca/bc/gov/educ/studentdatacollection/api/controller/v1/SdcSchoolCollectionStudentControllerTest.java

```diff
@@ -48,6 +49,13 @@ class SdcSchoolCollectionStudentControllerTest extends BaseStudentDataCollection
     @Autowired
     RestUtils restUtils;
 
+    @AfterEach
+    void cleanup(){
+        sdcSchoolCollectionStudentRepository.deleteAll();
+        sdcSchoolCollectionRepository.deleteAll();
+        collectionRepository.deleteAll();
```

Comment:
I think we can put all the database clean up methods into here https://github.com/bcgov/EDUC-STUDENT-DATA-COLLECTION-API/blob/master/api/src/test/java/ca/bc/gov/educ/studentdatacollection/api/support/StudentDataCollectionTestUtils.java 

^^^ it gets run after each test.